src/Native/gc/gcwks.cpp

```diff
@@ -13,6 +13,12 @@
 #include "gcdesc.h"
 #include "softwarewritewatch.h"
 
+#include "slist.h"
```

Comment:
Could you please add these includes to `src/Native/Runtime/gcenv.h` instead?

Reply:
We do not want to be including runtime-specific files from the GC directly to make the GC sources sharable across different runtimes. `gcenv.h` is the mechanism to allow it.

Reply:
Thank you! Why aren't include guards used in slist.h, RWLock.h and some others?

Reply:
The original folks working on the native runtime project did not like include guards. They preferred to have explicit list of all includes everywhere. (I did not really understand myself why it was a good thing.)

Reply:
ok, thank you!

Reply:
This convention is not consistently followed anymore. You might have notice that some of the files do have include guards already. I would not have a problem with adding them to more files as necessary.